frontend/src/pages/chat/Chat.tsx

```diff
@@ -544,8 +544,8 @@ const Chat = () => {
                                     className={styles.chatIcon}
                                     aria-hidden="true"
                                 />
-                                <h1 className={styles.chatEmptyStateTitle}>Start chatting</h1>
-                                <h2 className={styles.chatEmptyStateSubtitle}>This chatbot is configured to answer your questions</h2>
+                                <h1 className={styles.chatEmptyStateTitle}>{process.env.CHAT_TITLE || 'Start chatting'}</h1>
```

Comment:
Does this work when deployed? I'd be surprised that the frontend could access process.env, but perhaps I don't understand fully how the JSX compilation happens. I would assume the variables would need to get passed directly into the HTML templates for them to be accessible.

Reply:
@pamelafox have you tried this already? I have a customer that needs to change the title (Azure AI) to something more meaningful for their scenarios, and it would be great if they could just do this via an environment variable.

Haven't been able to test this PR yet, but I would assume that this environment variable is only present during compilation. Perhaps we can add such a feature via the API + states.

Reply:
I haven't tested this PR, no. Do report back if you test it.